docs/source/tutorials/quickstart.md

```diff
@@ -61,8 +62,8 @@ $ manim scene.py SquareToCircle -pl
 After some output, `manim` should render the scene into a .mp4 file, and open
 that file with the default movie player application.  You should see a video
 playing the following animation.
-
-![first scene output](_static/quickstart/first_scene.gif)
+/home/jan-hendrik/projects/manim-community/docs/source/_static/quickstart/first_scene.gif
```

Comment:
Why is this URL here?

Reply:
sorry, that was only for testing, forgot to remove it.